meshtastic/interfaces/ble/lifecycle_primitives.py

```
@@ -0,0 +1,417 @@
+        if safe_cleanup is not None:
+            try:
+                try:
+                    safe_cleanup(func=_tracked_cleanup, cleanup_name=operation_name)
+                except TypeError as exc:
+                    if not (
+                        _is_unexpected_keyword_error(exc, "func")
+                        or _is_unexpected_keyword_error(exc, "cleanup_name")
+                    ):
+                        raise
+                    safe_cleanup(_tracked_cleanup, operation_name)
+                return
```

Ruff (0.15.6)

[warning] Consider moving this statement to an `else` block

(TRY300)